pkg/loghttp/push/push_test.go

```diff
@@ -292,6 +308,37 @@ func TestParseRequest(t *testing.T) {
 			expectedBytesUsageTracker: map[string]float64{"{environment=\"dev\", foo=\"bar2\"}": float64(len("fizzbuzz")), "{environment=\"prod\", foo=\"bar2\"}": float64(len("xx")), "{dass=\"zasss\", foo=\"bar2\"}": float64(len("graf"))},
 			expectedLabels:            []labels.Labels{labels.FromStrings("foo", "bar2", "environment", "dev"), labels.FromStrings("foo", "bar2", "environment", "prod"), labels.FromStrings("foo", "bar2", "dass", "zasss")},
 		},
+		{
+			path: `/loki/api/v1/push`,
+			body: snappyString(marshalProto(
+				&logproto.PushRequest{
+					Streams: []logproto.Stream{
+						{
+							Labels: `{ "foo"="bar2",      "environment"="prod"   }`, // extra spaces are important
```

Comment:
I'm assuming the extra spaces are important for the assertion on stream label size? I think that's worth adding to the comment